external/include/mbedtls/config.h

```diff
@@ -2627,7 +2627,12 @@
 #undef MBEDTLS_ERROR_STRERROR_DUMMY
 #undef MBEDTLS_GENPRIME
 
-#undef MBEDTLS_FS_IO
+/* MBEDTLS_FS_IO is required for CURL.
```

Comment:
How about use ```#ifdef CONFIG_ENABLE_CURL```?
It makes easy to know why we block this.

Reply:
I agree. Will do the change.